labapi/src/main/java/com/microsoft/identity/internal/test/labapi/ApiClient.java

```diff
@@ -76,12 +77,20 @@ public class ApiClient {
 
     private HttpLoggingInterceptor loggingInterceptor;
 
+    /**
+     * No-parameter constructor will use default Base Path.
+     */
+    public ApiClient() {
+        this(DEFAULT_BASE_PATH);
+    }
+
     /*
      * Constructor for ApiClient
      */
-    public ApiClient() {
+    public ApiClient(final String basePathParam) {
```

Comment:
>public ApiClient(final String basePathParam) { [](http://example.com/codeflow?start=4&length=46)

IIRC, this file was auto generated by swagger code generator.
If we ever regenerate the code using that tool, how do we ensure that we persist these custom edits.

Reply:
What file is used to generate this one? I'm not seeing it. I was able to rebuild project and keep my changes, not sure if this means it's a static file, and not one generated at build time.

Reply:
I think this might only apply to Configuration.java, i don't see the annotation on ApiClient

Reply:
if you look at the top of this file it says.

 NOTE: This class is auto generated by the swagger code generator program.

Reply:
The instructions for codegen are defined here, you could update the instructions.

[https://github.com/AzureAD/android-complete/blob/master/docs/ProjectBuild/swagger-codegen.md](https://github.com/AzureAD/android-complete/blob/master/docs/ProjectBuild/swagger-codegen.md)

Reply:
I'll rework the code to not have to edit these, since it will require lab team to also edit their json

Reply:
Per teams chat, we haen't used code generation for a while for this part of our library. We'll need to create an ask on labs to update their swagger json, to update with regards to the new api. For now, we'll edit these files manually.